client/src/components/ErrorSnackbar.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
+import Snackbar from '@material-ui/core/Snackbar';
+
+const ErrorSnackbar = ({ error }) => {
+  const [state, setState] = useState({
+    open: false,
+    vertical: 'bottom',
+    horizontal: 'center',
+  });
+
+  const { vertical, horizontal, open } = state;
+
+  // const handleClick = (newState) => () => {
+  //   setState({ open: true, ...newState });
+  // };
```

Comment:
Try not to commit any commented out code in your PRs (generally bad practice)

Reply:
Removed this component as I'm no longer using it